tests/test_transformers.py

```diff
@@ -39,6 +39,13 @@ def test_df_selector_with_multiple_columns(categorical):
     assert {'category_a', 'category_b'} == set(result.columns)
 
 
+def test_df_selector_raise_missing_column(categorical):
+    select = Select(['category_a', 'category_b', 'category_c'])
+
+    with pytest.raises(TransformerError):
```

Comment:
Here we can add a `match=` argument to pytest.raises - that way we can check that the set logic works as expected and the error message appears as we expect :smiley: 
```suggestion
    with pytest.raises(TransformerError, match="The DataFrame does not include the columns: category_c):
```

Reply:
Smart!